### PR TITLE
improvement(result_analyze): Added email_subject_postfix to regression check

### PR DESCRIPTION
**Pull request by yarongilor**
in order to support various postfixes on performance email,
	this fix allow adding ending text like 'Cloud backend'

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 from sdcm import nemesis, cluster_docker, cluster_baremetal, cluster_k8s, db_stats, wait
 from sdcm.cluster import NoMonitorSet, SCYLLA_DIR, Setup, UserRemoteCredentials, set_duration as set_cluster_duration, \
-    set_ip_ssh_connections as set_cluster_ip_ssh_connections, BaseLoaderSet, BaseMonitorSet, BaseScyllaCluster
+    set_ip_ssh_connections as set_cluster_ip_ssh_connections, BaseLoaderSet, BaseMonitorSet, BaseScyllaCluster  # pylint: disable=no-name-in-module
```
Comment (bentsi) [inline sdcm/tester.py:38]: why did you add this? I don't see any issue here in `master`

Comment (yarongilor) [reply sdcm/tester.py:38]: the pre-commit fails for this in what looks like a python 3.8 issue:
```
pylint...................................................................Failed
- hook id: pylint
- exit code: 2

************* Module sdcm.tester
sdcm/tester.py:37:0: E0611: No name 'cluster' in module 'sdcm' (no-name-in-module)
sdcm/tester.py:37:0: E0001: Cannot import 'sdcm.cluster' due to syntax error 'invalid syntax (<unknown>, line 1514)' (syntax-error)

```

Comment (fruch) [reply sdcm/tester.py:38]: @yarongilor  is that error from a local run of your, or from Jenkins ?

**Review by fruch — CHANGES_REQUESTED** [review]
we won't be able to send it to SCT, you didn't defined `email_subject_postfix` in `sct_config.py`

**Comment by yarongilor** [comment]
> email_subject_postfix

already defined: 
https://github.com/scylladb/scylla-cluster-tests/blob/3103f8b23ab8e95f19089ab49fb4f45e51c33858/sdcm/sct_config.py#L324

**Review by fruch — APPROVED** [review]
LGTM
